pepper-apis/src/main/java/org/broadinstitute/ddp/service/actvityinstancebuilder/block/question/impl/BoolQuestionCreator.java

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.ddp.service.actvityinstancebuilder.block.question.impl;
+
+import org.broadinstitute.ddp.model.activity.definition.question.BoolQuestionDef;
+import org.broadinstitute.ddp.model.activity.instance.answer.BoolAnswer;
+import org.broadinstitute.ddp.model.activity.instance.question.BoolQuestion;
+import org.broadinstitute.ddp.service.actvityinstancebuilder.ActivityInstanceFromActivityDefStoreBuilder;
+import org.broadinstitute.ddp.service.actvityinstancebuilder.ElementCreator;
+import org.broadinstitute.ddp.service.actvityinstancebuilder.block.question.QuestionCreator;
+
+import static org.broadinstitute.ddp.service.actvityinstancebuilder.block.question.QuestionUtil.isReadOnly;
+
+/**
+ * Creates {@link BoolQuestion}
+ */
+public class BoolQuestionCreator extends ElementCreator {
+
+    public BoolQuestionCreator(ActivityInstanceFromActivityDefStoreBuilder.Context context) {
+        super(context);
+    }
+
+    public BoolQuestion createBoolQuestion(QuestionCreator questionCreator, BoolQuestionDef questionDef) {
```

Comment:
For simple question types like these, perhaps we can move these construction methods into `QuestionCreator` itself? I'll rather have a bigger class file than a ton of tiny files.